src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -246,6 +247,34 @@ getTsqlTvfType(Archive *fout, const FuncInfo *finfo, char prokind, bool proretse
 	return PLTSQL_TVFTYPE_NONE;
 }
 
+void fixAttoptionsBbfOriginalName(Archive *fout, char **attoptions)
+{
+	PGresult *res;
+	PQExpBuffer q;
+	int temp_buf_len;
+	unsigned char *temp_buf;
+
+	q = createPQExpBuffer();
+
+	appendPQExpBuffer(q,
```

Comment:
Could you add some comments explaining how this sql works? It would be nice if we can see some concrete examples.